daisy_workflows/image_import/enterprise_linux/translate.py

```diff
@@ -285,10 +351,31 @@ def yum_install(g, *packages):
       packages, last_exception))
 
 
+def customize_error_message(e: BaseException,
+                            errs: NonFatalErrors) -> BaseException:
+  """Create an error message based on PotentialErrors."""
+  logging.debug('Translation failed due to: {}'.format(e))
+  logging.debug('Potential errors: {}'.format(errs))
```

Comment:
Potential causes?

Reply:
Removed this log in the current version (I liked your idea of not using the `NonFatalErrors` class)